postgres/src/main/java/org/jdbi/v3/postgres/PGObjectColumnMapper.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.postgres;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.jdbi.v3.core.mapper.ColumnMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.postgresql.PGConnection;
+import org.postgresql.util.PGobject;
+
+/**
+ * Column mapper for {@link PGobject}.
+ */
+public class PGObjectColumnMapper implements ColumnMapper<PGobject> {
+
+    @Override
+    public PGobject map(ResultSet rs, int columnNumber, StatementContext ctx) throws SQLException {
+        PostgresTypes.addTypesToConnection((PGConnection) ctx.getConnection());
+
+        return (PGobject) rs.getObject(columnNumber);
```

Comment:
I think `SuppressWarnings` is in order around here too

Reply:
Could use `rs.getObject(columnNumber, PGobject.class)` instead